### DISCUSSION TITLE
Batcher support!

### DISCUSSION THREAD

**Discussion by mpourranjbar** [kubeflow/kfserving#2031] [Q&A]
Hi there,
Does Kserve support batch predictions when the cluster is installed without knative? I used the "Kubernetes Deployment Installation" method and I didn't install Knative. When I'm trying to set batch size I receive the following error. 
`required key CONTAINER_CONCURRENCY missing value`

**Comment by yuzisun** [comment]
@mpourranjbar the fix will be released shortly in 0.8

**Reply by yuzisun** [reply]
you can already try out with the pre-release v0.8.0-rc0

**Reply by mpourranjbar** [reply]
Than you @yuzisun for your quick response. 
I tried the pre-release v0.8.0-rc0 but it doesn't seem to solve that problem. Actually it introduces a new one. 
I have downloaded https://github.com/kserve/kserve/releases/download/v0.8.0-rc0/kserve.yaml and changed the `"defaultDeploymentMode"` to `"RawDeployment"`
But then I get the following error while trying to install kserve
`unable to recognize "kserve.yaml": no matches for kind "ClusterServingRuntime" in version "serving.kserve.io/v1alpha1"`
What did I miss?
